Resources/DesktopModules/Admin/Dnn.PersonaBar/Modules/Dnn.Prompt/App_LocalResources/prompt.nl-NL.resx

```diff
@@ -12,1095 +12,10 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="CommandNotFound.Text" xml:space="preserve">
```

Comment:
Tere are 1089 removed line in this file, is this on purpose ?

Reply:
Yes,  we used the translator tool from Peter Donker for the prompt cleaning and reviewed that manually comparing it with the en-US version of prompt.
All promote text elements are move to the PB modules and are no longer in the prompt .resx file itself.

Reply:
@aspelt Can you confirm the removal of a lot of text elements in prompt resx?